guides/advanced_guides/binary.md

```diff
@@ -1,86 +1,114 @@
-# MeiliSearch Binary
+# Installation
 
-## Downloading from Curl
+## Download and launch
+:::: tabs
 
-This script will download the **latest stable release** of MeiliSearch.
+::: tab cURL
+Download the **latest stable release** of MeiliSearch with **curl**.
+
+Launch the MeiliSearch package to start MeiliSearch's server.
 ```bash
 $ curl -L https://install.meilisearch.com | sh
 $ ./meilisearch
 Server is listening on: http://127.0.0.1:7700
 ```
+:::
 
-## Downloading with Homebrew
+::: tab Brew
+Download the **latest stable release** of MeiliSearch with **Homebrew**.
 
-The MeiliSearch binary (latest stable release) can be downloaded with the Homebrew package manager.
+Launch the MeiliSearch package to start MeiliSearch's server.
 ```bash
-$ brew install meilisearch
+$ brew update && brew install meilisearch
 $ meilisearch
 Server is listening on: http://127.0.0.1:7700
 ```
+:::
+
+::: tab Docker
+Using **Docker** you can choose to run [any available tags](https://hub.docker.com/r/getmeili/meilisearch/tags).
+
+This command line starts the **latest stable release** of MeiliSearch.
+```bash
+$ docker run -it --rm -p 7700:7700 getmeili/meilisearch
+Server is listening on: http://0.0.0.0:7700
+```
+:::
 
-## Downloading with APT
+::: tab APT
 
-The MeiliSearch binary (latest stable release) can be downloaded with the APT package manager.
+Download the **latest stable release** of MeiliSearch with **APT**.
+
+Launch the MeiliSearch package to start MeiliSearch's server.
 ```bash
 $ echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
 $ apt update && apt install meilisearch-http
 $ meilisearch
 Server is listening on: http://127.0.0.1:7700
 ```
+:::
 
-## Compiling from source
-
-MeiliSearch is made in `Rust`. Therefore the Rust toolchain must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
+::: tab Heroku
 
-If you have the Rust toolchain already installed, you need to clone the repository and go to the cloned directory.
+You can deploy the latest stable build of MeiliSearch straight on Heroku.
 
-```bash
-git clone https://github.com/meilisearch/MeiliSearch
-cd MeiliSearch
-```
+<p align="center">
+  <a href="https://heroku.com/deploy?template=https://github.com/meilisearch/MeiliSearch">
+    <img src="https://www.herokucdn.com/deploy/button.svg" alt="Deploy">
+  </a>
+</p>
 
-Inside the folder, compile MeiliSearch.
 
-```bash
-# Production version
-cargo build --release
+The deploy can take up to 20 minutes because it will compile the whole project from the GitHub repository.
```

Comment:
I'm not sure this is the point of this PR, but can we add a warning to say the data is not consistent on Heroku?